client/src/services/graphqlClient.ts

```diff
@@ -0,0 +1,255 @@
+import { ApolloClient, InMemoryCache, useQuery } from "@apollo/client";
+import gql from "graphql-tag";
+import { shallowEqual, useSelector } from "react-redux";
+import { RootState } from "../reducers";
+
+export const healthClient = new ApolloClient({
+  uri: "https://api.thegraph.com/index-node/graphql",
+  cache: new InMemoryCache(),
+});
+
+export const goerliClient = new ApolloClient({
```

Comment:
This is awesome! One thing I was thinking is we could maybe move the clients/queries/requests into separate files? Since we will be adding additional clients for the rounds